cmd/notation/version.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"runtime"
+	"strings"
+
+	"github.com/notaryproject/notation/internal/version"
+	"github.com/spf13/cobra"
+)
+
+func versionCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Show the notation version information",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runVersion()
+		},
+	}
+
+	return cmd
+}
+
+func runVersion() error {
```

Comment:
error is always nil, why do we need to return it.

Reply:
Thanks, Updated.